lib/test/test_hamburger.rb

```diff
@@ -0,0 +1,136 @@
+require_relative '../../lib/cdo/hamburger'
+require 'minitest/autorun'
+
+class HamburgerTest < Minitest::Test
+  def test_level_teacher_en
+    visibility = Hamburger.get_visibility({level: true, user_type: "teacher", language: "en"})
+
+    assert visibility[:hamburger_class]         == "show-always"
+    assert visibility[:show_teacher_options]    == "show-always"
+    assert visibility[:show_student_options]    == "hide-always"
+    assert visibility[:show_signed_out_options] == "hide-always"
+    assert visibility[:show_pegasus_options]    == "show-always"
+    assert visibility[:show_help_options]       == "show-always"
+  end
+
+  def test_level_teacher_nonen
+    visibility = Hamburger.get_visibility({level: true, user_type: "teacher", language: "fr"})
```

Comment:
And if you want to rotate, I also like to test with 'ar' for right to left languages. And then that covers a good amount for international.